### PR TITLE
Make single instance stuff work on Linux too

### PR DESCRIPTION
**Pull request by ConorIA**
This should address #75. Is there any reason why you don't want to single instance stuff to work on Mac? (i.e. Why was it Windows-only?)

~~EDIT: Oops. Doesn't seem to have done the trick.~~ There was a command missing. Works now.

### CODE REVIEW

**Comment by lipis** [comment]
The comment right above the if explains exactly that... and on macOS it's only one instance anyway.. that's how all apps are working :)

File: electron/main.js [hunk 1]
```diff
@@ -86,6 +86,9 @@ if (process.platform === 'win32') {
   });
 }
 
+if (shouldQuit) {
+  app.quit();
+}
```
Comment (ConorIA) [inline electron/main.js]: It seems to be working ok on my Manjaro box. What about something like: 

```
if (process.platform === 'linux' && shouldQuit) {
  app.quit();
}
```

Should that be a single `&` or double `&&` in JS?

Comment (lipis) [inline electron/main.js]: I would also put it inside the above `if` but again.. we'll have to investigate that more.. for example if the app is minimized.. will it show the current instance that is running when you try to open a new one?

File: electron/main.js [hunk 2]
```diff
@@ -84,9 +84,11 @@ if (process.platform === 'win32') {
     }
     return true;
   });
+  if (process.platform === 'linux' && shouldQuit) {
+    app.quit();
+  }
 }
 
-
```
Comment (lipis) [inline electron/main.js]: leave that untouched :)

Comment (ConorIA) [inline electron/main.js]: Sorry, I thought it was a line I had added by mistake!

**Comment by lipis** [comment]
Thank you!

**Pull request merged**